docker/Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM python:3.8.2-buster
+FROM python:3.11-bookworm
```

Comment:
Are there plans to migrate the Dockerfiles to AlmaLinux9? `bookworm` sounds like a Debian derivative. TSSW uses AlmaLinux with Mamabaforge for Python and conda.

Reply:
I checked with @tribeiro and this is not a priority yet, but I extended a ticket description I had for this purpose to state we expect to use AlmaLinux too :+1: [DM-44923](https://rubinobs.atlassian.net/browse/DM-44923).

Reply:
Thanks for the info!